src/rgw/rgw_data_sync.cc

```diff
@@ -517,74 +520,67 @@ bool RGWListRemoteDataLogCR::spawn_next() {
 }
 
 class RGWInitDataSyncStatusCoroutine : public RGWCoroutine {
-  static constexpr uint32_t lock_duration = 30;
-  RGWDataSyncCtx *sc;
-  RGWDataSyncEnv *sync_env;
-  rgw::sal::RadosStore* store; // RGWDataSyncEnv also has a pointer to store
-  const rgw_pool& pool;
+  static constexpr auto lock_name{ "sync_lock"sv };
+  static constexpr uint32_t lock_duration{ 30 };
```

Comment:
why is this hardcoded to 30 sec and not taken from `rgw_sync_lease_period` (which defaults to 2 min)?

Reply:
I don't know, but I'll change it.